benchmark/blankenbach/readme.txt

```diff
@@ -0,0 +1,50 @@
+Blankenback benchmark, see table 9 for reference values
+
+Output:
+# Nu           Vrms           name:
+4.78661864e+00 4.34590432e+01 case1a_ref4.stat
```

Comment:
The comment at the top doesn't give much away -- are these *our* results, or are they the ones from the paper?